apps/worker/src/app/workflow/services/standard.worker.ts

```diff
@@ -59,22 +55,26 @@ export class StandardWorker extends StandardWorkerService implements INovuWorker
     };
   }
 
-  private extractMinimalJobData(job: any): {
+  private extractMinimalJobData(data: IStandardDataDto): {
     environmentId: string;
     organizationId: string;
     jobId: string;
     userId: string;
   } {
-    const { _environmentId: environmentId, _id: jobId, _organizationId: organizationId, _userId: userId } = job;
+    const { _environmentId: environmentId, _id: jobId, _organizationId: organizationId, _userId: userId } = data;
 
     if (!environmentId || !jobId || !organizationId || !userId) {
-      const message = job.payload.message;
+      const message = data.payload?.message;
```

Comment:
This object is structure `data.payload?.message` deprecated, i could not locate any place in the code that passes this data.